py/desiutil/redirect.py

```diff
@@ -0,0 +1,247 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+# -*- coding: utf-8 -*-
+"""
+===================
```

Comment:
Please make sure that this doc string is valid RST.  I notice the `=` do not match the length of the title.

Reply:
Fixed